docs/development/manufacturer/manufacturer-design-guidelines.md

```diff
@@ -589,7 +590,7 @@ Define correct flash driver(s) only if physical present on the board.
 
 ### 4.2.3 Defines for BARO
 
-Define a barometer only if physical present on the board.
+Define a barometer only if physical present on the board. Betaflight strongly recommends I2C for barometer connections.
```

Comment:
Different I2C for internal and external devices ?

Reply:
Or at least have the onboard baro's I²C address configurable with e.g. a solder jumper. Not having different I²C busses for internal/external and not having a configurable address should nog be allowed IMO.